package.json

```diff
@@ -8,18 +8,18 @@
     "npm": "^10.3.0"
   },
   "scripts": {
+    "build": "npm run decrypt-service-account && npm run download-ics && npm run get-events && gatsby build",
     "pr-build": "gatsby build",
-    "build": "npm run setup-environment && npm run download-ics && npm run get-events && gatsby build",
     "develop": "npm run get-mock-events && gatsby develop",
     "start": "gatsby develop",
     "serve": "gatsby serve",
-    "clean": "gatsby clean",
+    "clean": "gatsby clean && npx rimraf dist",
```

Comment:
Since we have `rimraf` as a dependency, should this be `npm` instead of `npx`? Same question with line 22 below.